Apigee/SmartDocs/Revision.php

```diff
@@ -301,12 +306,15 @@ public function toArray($verbose = TRUE)
      *
      * @param OrgConfig $config
      * @param string $modelId
+     * @param string $modelName
      */
-    public function __construct(OrgConfig $config, $modelId)
+    public function __construct(OrgConfig $config, $modelId, $modelName = "")
     {
         $this->blankValues();
         $this->apiId = $modelId;
-        $this->init($config, '/o/' . rawurlencode($config->orgName) . '/apimodels/' . rawurlencode($this->apiId) . '/revisions');
+        $this->apiName = $modelName;
+//        $this->init($config, '/o/' . rawurlencode($config->orgName) . '/apimodels/' . rawurlencode($this->apiId) . '/revisions');
```

Comment:
Guilty.  I removed /code rawurlencode($this->apiId) . '/revisions' /code from this line and added it elsewhere to accommodate the path of the new endpoint.  I kept it in so I could easily copy and paste it, but yeah, no reason to keep it in.  I'll remove it.